contrib/sirupsen/logrus/example_test.go

```diff
@@ -29,6 +29,6 @@ func ExampleHook() {
 	cLog := logrus.WithContext(sctx).WithTime(time.Date(2000, 1, 1, 1, 1, 1, 0, time.UTC))
 	// Log as desired using the context-aware logger
 	cLog.Info("Completed some work!")
-	// Output:
+	// You should see:
```

Comment:
I'm curious the motivation of the change, was the testable example failing? I'm wondering if we actually run these in CI :thinking:

Reply:
They run as part of `go test` normal behaviour. It didn't work because the hook is not adding the fields for trace ID and span ID.

I checked the hook's code and it wasn't changed. I'm not sure why it triggered here, but it did.